### PR TITLE
Calculator unit tests

### PR DESCRIPTION
**Pull request by jnicolaslaurenti**
The calculator passed the tests successfully

Tests:

image

Coverage:

image

### CODE REVIEW

File: app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        verify(view).resetResultView()
+        verify(view).resetOperationView()
+        verifyNoMoreInteractions(view)
+    }
+
+    @Test
+    fun ErrorIncompleteOperation() {
+        model.setValue(ONE_STRING)
+        model.setOperator(DIV)
+        val error = model.getError()
```
Comment (SartiMau) [inline app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt]: this variable is not necessary, you can remove it and modify the assertEquals to use model.getError()

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt (hunk 1 above)
Comment (jnicolaslaurenti) [reply app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt]: According to what was discussed, it was decided not to change it

File: app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+        presenter.onCleanButtonPressed()
+        verify(view).showCompleteOperation(model.getOperation())
+        verify(view).showLastValue(model.getLastModified())
+        verify(view).showDeleteMessage()
+        assertEquals(OPERATION_CLEAN_A_VALUE_IN_SECOND_OPERAND, model.getOperation())
+        verifyNoMoreInteractions(view)
+    }
+
+    @Test
+    fun CleanAValueInFirstOperand() {
```
Comment (mauroLonghi) [inline app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt]: PLs change to lowerCase all the Test methods

Comment (jnicolaslaurenti) [reply app/src/test/java/com/onboarding/calculatorkotlin/CalculatorTest.kt]: Solved!

File: app/src/main/java/com/onboarding/calculatorkotlin/mvp/model/CalculatorModel.kt [hunk 3]
```diff
@@ -66,20 +66,21 @@ class CalculatorModel : CalculatorContract.Model {
...
         if (!operandEnabled(firstOperand) || !operandEnabled(secondOperand)) {
-            return Error.ERROR_INCOMPLETE_OPERATION
+            return Result.ERROR_INCOMPLETE_OPERATION
         }
         if (operandEnabled(firstOperand) && (operator == DIV) && (secondOperand == ZERO_STRING)) {
-            return Error.ERROR_DIVISION_BY_ZERO
+            return Result.ERROR_DIVISION_BY_ZERO
         }
-        return Error.NONE
+        return Result.SUCCES
```
Comment (lucianoeli) [inline app/src/main/java/com/onboarding/calculatorkotlin/mvp/model/CalculatorModel.kt]: it spels SUCCESS

**Review by lucianoeli — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**